profiles/apparmor.d/usr.share.openqa.script.worker

```diff
@@ -165,3 +167,51 @@
 
   }
 }
+
+  profile /usr/bin/Xvnc {
```

Comment:
these profiles are duplicated - can we close this PR?

Reply:
Sure, it was here just to have a better insight of the proposed changes in the progress issue